schedule/yam/agama_cryptlvm.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
let's always use more readable names, agama_lvm_encrypted.yam.
Same for the other files.